index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Kindly update the readme using this [template](https://github.com/microverseinc/readme-template).

Reply:
@mikenath223 changes made